src/EditorFeatures/CSharp/EmbeddedLanguages/CSharpJsonBraceMatcher.cs

```diff
@@ -12,13 +12,12 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.EmbeddedLanguages
 {
     [ExportEmbeddedLanguageBraceMatcherInternal(
-        PredefinedEmbeddedLanguageBraceMatcherNames.Json, LanguageNames.CSharp, supportsUnannotatedAPIs: true, "Json"), Shared]
-    internal sealed class CSharpJsonEmbeddedLanguageBraceMatcher :
-        AbstractJsonEmbeddedLanguageBraceMatcher
+        PredefinedEmbeddedLanguageNames.Json, LanguageNames.CSharp, supportsUnannotatedAPIs: true, "Json"), Shared]
+    internal sealed class CSharpJsonBraceMatcher : AbstractJsonBraceMatcher
```

Comment:
instead of having names for each embeded-language/feature-name pair, we just have the names of hte embedded languages once.